pkgs/applications/office/libreoffice/generate-libreoffice-srcs.py

```diff
@@ -8,28 +8,61 @@
 
 todo - Ideally we would move as much as possible into derivation dependencies.
 """
-import collections, itertools, json, re, sys, os
+import collections, itertools, json, re, subprocess, sys, os
 
 def main():
 
+    packages = list(get_packages())
+
+    for x in packages:
+        print(x, file=sys.stderr)
+
     print('[')
 
-    for x in get_packages():
+    for x in packages:
 
-        print('{')
+        md5 = x['md5']
+        tarball = x['tarball']
 
-        print('  name = "{}";'.format(x['tarball']))
-        print('  md5 = "{}";'.format(x['md5']))
-        print('  brief = {};'.format('true' if x['brief'] else 'false'))
+        url = (
+            'http://dev-www.libreoffice.org/src/{}{}'.format(
+                x.get('subdir', ''), tarball)
+            if x['brief'] else
```

Comment:
Infix `if`? Why not normal prefix `if`? I think indentation pattern will be more readable for people not writing Python/Ruby/Perl as their main language…